plugins/user-settings/src/components/FeatureFlags/UserSettingsFeatureFlags.tsx

```diff
@@ -99,9 +106,6 @@ export const UserSettingsFeatureFlags = () => {
     <Grid container style={{ justifyContent: 'space-between' }}>
       <Grid item xs={6} md={8}>
         <Typography variant="h5">Feature Flags</Typography>
-        <Typography variant="subtitle1">
-          Please refresh the page when toggling feature flags
-        </Typography>
```

Comment:
The feature flag system is not built to support runtime changes to feature flags. There might be some instances where it works, but often a full page reload is needed. To implement this correctly we'd need some form of metadata attached to the flag that indicates whether it supports runtime changes or not.

Reply:
Ah, that makes sense. Might need to do some digging to find a way to surface that metadata to achieve this in a way that provides nice feedback to user, but also takes runtime/!runtime flags into consideration.